build/workflow/pipeline.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         DisableNet6MobileTargets: true
 
   pool:
-    vmImage: 'macOS-11'
+    vmImage: 'macos-12'
```

Comment:
```suggestion
    vmImage: 'macOS-12'
```
Should be macOS instead of macos @kazo0 no?

Reply:
yes :)